controller/appcontroller.go

```diff
@@ -78,13 +79,18 @@ const (
 type CompareWith int
 
 const (
-	// Compare live application state against state defined in latest git revision with no resolved revision caching.
+	// CompareWithLatestForceResolve compares live application state against state defined in latest git revision with
+	// no resolved revision caching. This is the default level when the user manually requests a refresh.
 	CompareWithLatestForceResolve CompareWith = 3
-	// Compare live application state against state defined in latest git revision.
+	// CompareWithLatest compares live application state against state defined in latest git revision.
```

Comment:
This looks pretty similar to `CompareWithLatestForceResolve` but this is probably using cache? Should we add that info to make it more descriptive because without looking at the full list it would be difficult to know of that behaviour.

Reply:
```suggestion
	// CompareWithLatest compares live application state against state defined in latest git revision, according
	// to the git revision cache.
```

Maybe something like that?

Reply:
Looks good!